RELEASE_NOTES.md

```diff
@@ -1,5 +1,60 @@
 # Release Notes
 
+## 2.0.5
+### Features
+- Rootless Podman will now add an entry to `/etc/passwd` for the user who ran Podman if run with `--userns=keep-id`.
+- The `podman system connection` command has been reworked to support multiple connections, and reenabled for use!
+- Podman now has a new global flag, `--connection`, to specify a connection to a remote Podman API instance.
```

Comment:
thought/suggestion:  It might be good to include a pointer to the online man page for more information.